docs/how-to-guides/admin/privacycookie.rst

```diff
@@ -42,12 +42,38 @@ content. We use `cookie consent <https://github.com/Mastermindzh/react-cookie-co
 create the banner and we support customization of all the properties exposed by the library.
 All the adjustments should be provided as values in ``ui.privacy.banner`` as follows:
 
-1. ``content``: The HTML content of the cookie banner. Please mind that links should point
+1. ``content``: the HTML content of the cookie banner. Please mind that links should point
    to local content. Use the privacy page to add references to third party sites.
-2. ``layout``: The list of properties for the cookies consent plugin. We use
+2. ``layout``: the list of properties for the cookies consent plugin. We use
    `Bootstrap <https://getbootstrap.com/docs>`_ for the UI, therefore all the Bootstrap
    classes can be used to customize the appearance.
 
 You can refer to the current
 `values.yaml in the renku-ui repository <https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/values.yaml>`_
 to check the default values and to find an example of how to apply different settings.
+
+Dashboard message
+~~~~~~~~~~~~~~~~~
+
+The UI can display a configurable message for logged in users on their dashboard
+page, suited for showing general information about Renku.
+
+This feature can be enabled and configured by editing the values found in
+``ui.client.dashboardMessage``. Set ``ui.client.dashboardMessage.enabled``
+to ``true`` to have the message displayed on the dashboard page.
+
+The content and appearance of the message are configured by editing the other
+values in ``ui.client.dashboardMessage``:
+
+1. ``text``: the main message displayed on the dashboard page. It supports
+   `Markdown syntax <https://en.wikipedia.org/wiki/Markdown>`_.
+2. ``additionalText``: the additional message which users can read by clicking
+   "Read more" button right below the main message.
+3. ``styles``: the appearance of the card used to display the
```

Comment:
why is this called `styles` and not `style`? only setting one, presumably

Reply:
it's a typo